app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -172,14 +163,48 @@ def save(resource)
           false
         end
 
-        # switches between using valkyrie to save or active fedora to save
+        # copy visibility from the work resource to the file_set resource
+        def propagate_visibility(work, file_set)
```

Comment:
is this already handled by the asynchronous `VisibilityCopyJob`? or perhaps `ResourceVisibilityPropagator` should be called directly here?